packages/observe-sequence/observe_sequence_tests.js

```diff
@@ -45,11 +45,11 @@ runOneObserveSequenceTestCase = function (test, sequenceFunc,
 
       firedCallbacks.splice(i, 0, obj);
     },
-    removedAt: function () {
-      firedCallbacks.push({removedAt: _.toArray(arguments)});
+    removedAt: function (...args) {
+      firedCallbacks.push({removedAt: args});
     },
     movedTo: function () {
-      firedCallbacks.push({movedTo: _.toArray(arguments)});
+      firedCallbacks.push({movedTo: args});
```

Comment:
there is no mention to args in this function parameters

Reply:
Thanks for noticing these @renanccastro. You may review again now.